metals/src/main/scala/scala/meta/internal/metals/Messages.scala

```diff
@@ -296,6 +296,55 @@ object Messages {
     }
   }
 
+  object BloopGlobalJsonFilePremodified {
+    def applyAndRestart: MessageActionItem =
+      new MessageActionItem("Apply and Restart Bloop")
+    def saveButNotRestart: MessageActionItem =
```

Comment:
I think we don't need this option. If some one already modifies the jvm settings their intention is to restart it as well.

Reply:
ok I would remove it